tests/grid/grid_tools_aspect_ratio.cc

```diff
@@ -0,0 +1,234 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2001 - 2018 by the deal.II authors
```

Comment:
```suggestion
// Copyright (C) 2019 by the deal.II authors
```

Reply:
changed in https://github.com/dealii/dealii/pull/8894/commits/c860e2131a5647fd37eb07272b1b88df0a75a1df

It seems we are converged now?